tests/autobahn/test_autobahn.py

```diff
@@ -0,0 +1,38 @@
+import shutil
+from pathlib import Path
+
+import pytest
+from python_on_whales import DockerClient
+
+
+@pytest.fixture(scope="session", autouse=True)
+def create_report_directory():
```

Comment:
Instead of doing this in-tree, which may be problematic in parallel envs, consider using `tmp_path` fixture (pytest builtin) that provides you with an empty dir dedicated to your current test run.

Reply:
I didn't know about such a thing, thank you for the suggestion I will look into it.

Reply:
I couldn't figure out how to bind the created directory using `tmp_path` to the `crossbario/autobahn-testsuite` report volume.